invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -0,0 +1,232 @@
+              record.parent.access.settings.accept_conditions_text,
+            secret_link_expiration: this.translateExpirationDate(
+              record.parent.access.settings.secret_link_expiration
+            ),
+          }}
+          validationSchema={this.accessRequestSchema}
+        >
+          {({ values, handleSubmit }) => {
+            return (
+              <Grid>
+                <Form>
+                  <Grid>
+                    <Grid.Row>
+                      <Grid.Column width={16}>
+                        <Form.Field>
+                          <RadioField
+                            checked={_get(values, "allow_user_requests")}
+                            control={Checkbox}
+                            fieldPath="allow_user_requests"
+                            label={i18next.t(
+                              "Allow authenticated users to request access to restricted records/files."
+                            )}
+                            onChange={({ data, formikProps }) => {
+                              formikProps.form.setFieldValue(
+                                "allow_user_requests",
+                                data.checked
+                              );
+                            }}
+                          />
+                        </Form.Field>
+                        <Form.Field>
+                          <RadioField
+                            checked={_get(values, "allow_guest_requests")}
+                            control={Checkbox}
+                            fieldPath="allow_guest_requests"
+                            label={i18next.t(
+                              "Allow non-authenticated users to request access to restricted records/files."
+                            )}
+                            onChange={({ data, formikProps }) => {
+                              formikProps.form.setFieldValue(
+                                "allow_guest_requests",
+                                data.checked
+                              );
+                            }}
+                          />
+                        </Form.Field>
+                        <label className="helptext mb-0 mt-10">
+                          {i18next.t(
+                            "Enable users and guests to request access to your record. You will get an email " +
+                              "asking for approval. After you approve a request, users will be granted access and " +
+                              "guests will receive a secret link."
+                          )}
+                        </label>
+                      </Grid.Column>
+                    </Grid.Row>
+                    <Grid.Row>
+                      <Grid.Column>
+                        <Form.Field>
+                          <TextAreaField
+                            placeholder={i18next.t(
+                              "Optional. Specify conditions under which you approve access."
```

Comment:
```suggestion
                              "Optional. Specify conditions under which you approve access. This message will be visible by any user requesting access in the access request box."
```
I don't know if `access request box` is the right term